core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2980,6 +3110,22 @@ private void setAttributeFromEntry(SetAttributeEntry entry)
     return mBlockMaster.getWorkerInfoList();
   }
 
+  private void checkAndAppendAuditLog(RpcUtils.RpcContext context) {
+    if (!mAuditLog.isEnabled()) { return; }
+    mAuditLog.append(context);
+  }
+
+  private void checkAndCommitAuditLog(String command, String src, String dst, Inode inode, RpcUtils.RpcContext context) throws AccessControlException {
+    if (!mAuditLog.isEnabled()) { return; }
+    context.setCommand(command).setUser(AuthenticatedClientUser.getClientUser())
```

Comment:
it is possible for `AuthenticatedClientUser.getClientUser()` to throw an AccessControlException when the current client user is not set (i.e. null), will this cause the operation to fail and return an exception to the thrift client?